src/libecalc/common/tabular_time_series.py

```diff
@@ -50,9 +52,9 @@ def merge(cls, *objects_with_time_series: ObjectWithTimeSeries):
             for other in others:
                 accumulated_value = merged_object.__getattribute__(key)
                 other_value = other.__getattribute__(key)
-                if key == "timesteps":
-                    merged_timesteps = sorted(itertools.chain(accumulated_value, other_value))
-                    merged_object.__setattr__(key, merged_timesteps)
+                if key == "periods":
+                    merged_periods = sorted(itertools.chain(accumulated_value, other_value), key=attrgetter("start"))
```

Comment:
attrgetter may not be necessary when the Period(s) classes have `@dataclass(eq=True, frozen=True, order=True)`